src/js/components/Feedback/stories/FeedbackNegPos.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
In the grommet theme the spacing feels weird on this. I would expect the Icons to be aligned with the "Was this content helpful?" text and have a little spacing between the FormField underline.

<img width="334" alt="Screen Shot 2022-11-23 at 2 29 36 PM" src="https://user-images.githubusercontent.com/54560994/203649240-a097e56c-b478-4ad9-9be7-c3ab2f38edec.png">